src/chains/filecoin/types/src/data-managers/block-messages-manager.d.ts

```diff
@@ -0,0 +1,18 @@
+import Manager from "./manager";
+import { LevelUp } from "levelup";
+import { BlockMessages, BlockMessagesConfig } from "../things/block-messages";
+import { CID } from "../things/cid";
+import SignedMessageManager from "./message-manager";
+export default class BlockMessagesManager extends Manager<
+  BlockMessages,
+  BlockMessagesConfig
+> {
+  #private;
```

Comment:
does this do anything?

Reply:
Not that I know of, but everything under `src/chains/filecoin/types` is autogenerated by `ttsc` with the `npm run tsc.filecoin.declarations` command